film_scan_aip_documentation.txt

```diff
@@ -6,24 +6,24 @@ parent_directory - either OE (oe##### - Object_Entry Spectrum Collections Manage
 ├── uuid_directory - this directory contains the AIP. The identifier is  for the entire representation/package.
 │   ├── logs - logs that are generated via IFIscripts. This can include process logs for FFmpeg/rawcooked and other tools.
 │   │   └── uuid_sip_log.log - the most important log - this is a rolling log for all processes that IFIscripts will execute.
-│   ├── metadata - for AV objects - mediainfo and mediatrace XML files. For non-AV objects, Siegfried/PRONOM and EXIFTOOL JSON reports. Once the package is accessioned, there will be a Digital Forensics XML minus the hashes. This will also contain PBCore/Technical metadata CSV and IFI Filmographic CSV in the accessioned package.
+│   ├── metadata - for AV objects - mediainfo and mediatrace XML files. For non-AV objects, Siegfried/PRONOM and EXIFTOOL JSON reports. Once the package is aipped, there will be a Digital Forensics XML minus the hashes. This will also contain PBCore/Technical metadata CSV and IFI Filmographic CSV in the accessioned package.
```

Comment:
Not sure if aipped is best here. Especially as the noun is also package.

Reply:
@hyz2993 suggest rewriting to: 'Once the AIP is created it will contain a Digital Forensics XML minus the hashes. It will also contain PBCore/Technical metadata CSV and IFI Filmographic CSV.'

Reply:
Updated.